src/main/java/com/cloudcoders/gestaca/logic/enrollment/AddEnrollment.java

```
@@ -23,14 +24,22 @@ public AddEnrollment(IEnrollmentDAO iEnrollmentDAO,
 
   public void add(Enrollment enrollment) throws InvalidPersonException, InvalidTaughtCourse {
 
-    if (iStudentDAO.get(enrollment.getStudent().getId()) == null) {
-      throw new InvalidPersonException("Invalid person!");
+    try {
+      if (iStudentDAO.get(enrollment.getStudent().getId()) == null) {
+        throw new InvalidPersonException("Invalid person!");
+      }
+    } catch (PersistenceException e) {
+      e.printStackTrace(); //TODO handle exception properly
```

Need to propagate a unexpected exception